src/Microsoft.AspNetCore.Rewrite/ModRewrite/ModRewriteConditionPatternParser.cs

```diff
@@ -0,0 +1,229 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.AspNetCore.Rewrite.ModRewrite
+{
+    /// <summary>
+    /// Parses the "CondPattern" portion of the RewriteCond. 
+    /// RewriteCond TestString CondPattern
+    /// </summary>
+    public static class ModRewriteConditionPatternParser
```

Comment:
Now that ModRewrite is in the namespace do we need it in the type names still? Is there any danger of these types colliding with UrlRewrite or other rule systems?